### PR TITLE
nitpick: Update documentation and naming in read_table_list function

### LINKED ISSUE #11427: Composer sample bq_copy_across_locations.py misuses the word "tuple" in a function.
Thanks for stopping by to let us know something could be better!

**PLEASE READ**: If you have a support contract with Google, please create an issue in the [support console](https://cloud.google.com/support/) instead of filing on GitHub. This will ensure a timely response.

**The issue you're having must be related to a file in this repository.** We are unable to provide assistance for issues unrelated to samples in this repository.

Please include as much information as possible:

## In which file did you encounter the issue?

[https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/composer/workflows/bq_copy_across_locations.py#L107](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/composer/workflows/bq_copy_across_locations.py#L107)
## Did you change the file? If so, how?
No

## Describe the issue
The function documentation says it returns a list of tuples but it actually returns a list of dictionaries. There is a local variable that should be renamed as well.

Making sure to follow these steps will guarantee the quickest resolution possible.

Thanks!

### PR DESCRIPTION
**Pull request by Deseao**
Correct the documentation of the function to correctly specify its return value, and update a locally scoped variable to not refer to its type but instead its semantic meaning.

## Description

Fixes #11427 

Note: Before submitting a pull request, please open an issue for discussion if you are not associated with Google.

## Checklist
- [x] I have followed [Sample Guidelines from AUTHORING_GUIDE.MD](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md)
- [ ] README is updated to include [all relevant information](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#readme-file)
- [ ] **Tests** pass:   `nox -s py-3.9` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#test-environment-setup))
- [ ] **Lint** pass:   `nox -s lint` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#test-environment-setup))
- [ ] These samples need a new **API enabled** in testing projects to pass (let us know which ones)
- [ ] These samples need a new/updated **env vars** in testing projects set to pass (let us know which ones)
- [ ] This sample adds a new sample directory, and I updated the [CODEOWNERS file](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/.github/CODEOWNERS) with the codeowners for this sample
- [ ] This sample adds a new **Product API**, and I updated the [Blunderbuss issue/PR auto-assigner](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/.github/blunderbuss.yml) with the codeowners for this sample
- [x] Please **merge** this PR for me once it is approved

### CODE REVIEW

**Review by m-strzelczyk — APPROVED** [review]

**Comment by Deseao** [comment]
This needs another approver, is this blocked by anything on my end?

**Review by rsamborski — APPROVED** [review]

**Pull request merged**
